src/component/header/HeaderThree.jsx

```diff
@@ -6,7 +6,7 @@ import Scrollspy from 'react-scrollspy'
 const SocialShare = [
     {Social: <FaFacebookF /> , link: 'https://www.facebook.com/'},
     {Social: <FaLinkedinIn /> , link: 'https://www.linkedin.com/'},
-    {Social: <FaInstagram /> , link: 'https://www.instagram.com/'},
+    {Social: <FaInstagram /> , link: 'https://www.instagram.com/consult_a_criminal/?hl=en'},

```

Comment:
Good work with updating the link!

Looks like this issue is actually a 2 liner. We'll have to find where this data gets sent to. 

We're looking for an anchor tag  `<a>` somewhere here.